src/main/java/org/eclipse/uprotocol/client/Notifier.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * SPDX-FileCopyrightText: 2024 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.eclipse.uprotocol.client;
+
+import org.eclipse.uprotocol.v1.*;
```

Comment:
Not sure if there's a Java standard that's being followed, or even if there is an overarching "Java standard", but Google's java style guide says not to use wildcard imports:
https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports
I find code much easier to read without them, and of course there's the question of someone adding a new method to a package...

Reply:
fixed everywhere